packages/phpstan-rules/tests/Rules/PreventDuplicateClassMethodRule/Fixture/DifferentMethodName.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\PHPStanRules\Tests\Rules\PreventDuplicateClassMethodRule\Fixture;
+
+class DifferentMethodName1
+{
+    public function go($value)
+    {
+        $value += 100000;
+        return $value + 1000;
+    }
+}
+
+class DifferentMethodName2
+{
+    public function sleep($yet)
+    {
+        $yet += 100000;
+        return $yet + 1000;
+    }
+}
```

Comment:
For PHPStan it's different than for coding standard. These have to be split into 2 files so PSR-4 works.

Then provide both files in one tests, e.g. like here: https://github.com/symplify/symplify/blob/f7fbbca9a4eb388c53e0d62467a687c263fe462e/packages/phpstan-rules/tests/Rules/PreventDuplicateClassMethodRule/PreventDuplicateClassMethodRuleTest.php#L38-L40

Reply:
implemented.